### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Hexcede**

### CODE REVIEW

**Comment by MiniMeatwad** [comment]
LGTM!

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Voltz OSS
+
+An open-sourced Voltz remake.
```
Comment (4812571) [inline README.md]: Remember, the goal of this project is to be a logical continuation of Voltz, not a remake. (despite actually being a remake)

Comment (MiniMeatwad) [inline README.md]: Replace with "An open-sourced Voltz"

**Review by MiniMeatwad — COMMENTED** [review]
Replace

**Review by MiniMeatwad — APPROVED** [review]

**Review by 4812571 — APPROVED** [review]
LGTM!

**Pull request merged**